static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -1,6 +1,5 @@
 // @flow
-import { acquisitionsSupportUsRecurringContribution } from 'common/modules/experiments/tests/acquisitions-support-us-recurring-contributions';
 
 export const membershipEngagementBannerTests: $ReadOnlyArray<
     AcquisitionsABTest
-> = [acquisitionsSupportUsRecurringContribution];
+> = [];
```

Comment:
Do we need this module anymore?

Reply:
I'd be surprised if there aren't other tests due imminently. But if not, I guess it could go.

Reply:
After, talking with @joelochlann he thinks that there are going to be more test in the near future and deleting that array probably is not trivial.